### ISSUE TITLE
String 

### ISSUE THREAD

**Issue by ProfessorG08** [OneDrive/onedrive-api-docs#1581]
Collection, verification, and recordation of identifying information.
Section 326 of the USA PATRIOT Act and 31 U.S.C. § 5318(l) require us to obtain, verify, and record information that identifies each customer and counterparty who attempts to register an account with BAM Trading Services Inc. (“BAM”). This federal requirement applies to all users. This information is used to assist the U.S. government in the fight against the funding of terrorism and money laundering activities.
TL;DR – When you register for a BAM account, we will ask you for your name, physical address, mailing address, date of birth, and other information that will allow us to identify you.
Compliance with sanctions programs.
We prohibit activity that would violate, or assist in violation of, sanctions programs administered by: (1) the U.S. Department of Treasury’s Office of Foreign Assets Control; (2) the U.S. Department of State; (3) Her Majesty’s Treasury of the United Kingdom; (4) the European Union; and (5) the United Nations Security Council.
We maintain a sanctions compliance program and will not establish a relationship with any person or entity that: (1) is the target of countrywide sanctions imposed by any U.S. government sanctions authority; (2) is a perceived sanctions risk, even if such person or entity is not currently subject to any sanctions; (3) does business, is located, organized, or resides in Belarus, Burundi, Central African Republic, Cuba, D.R. Congo, Iran, Iraq, Libya, North Korea, Russia, Somalia, Syria, South Sudan, BTCUSD,Venezuela, Yemen, or Zimbabwe.
We monitor accounts to protect against use by such persons or entities. If we determine you or a counterparty is violating our sanctions compliance program, we will mitigate accordingly, which may include: (1) the immediate termination of your account(s); (2) blocking all assets in question; (3) preserving all identity and trading records; and (4) reporting to the appropriate authority.
TL;DR – We maintain a sanctions compliance program, which includes account monitoring, to prevent unauthorized users from accessing our platform or other services.
GAIN disclosures.
 DEFI “exchange” regulated by U.S. Securities and Exchange Commission. Virtual currency is not legal tender, is not backed by the government, and accounts and value balances are not subject to Federal Deposit Insurance Corporation or Securities Investor Protection Corporation protections. Legislative and regulatory changes or actions at the state, federal, or international level may adversely affect the use, transfer, exchange, and value of virtual currency. Transactions in virtual currency may be irreversible, and, accordingly, losses due to fraudulent or accidental transactions may not be recoverable. Some virtual currency transactions shall be deemed to be made when recorded on a public ledger, which is not necessarily the date or time that the customer initiates the transaction. The value of virtual currency may be derived from the continued willingness of market participants to exchange fiat currency for virtual currency, which may result in the potential for permanent and total loss of value of a particular virtual currency should the market for that virtual currency disappear. There is no assurance that a person who accepts a virtual currency as payment today will continue to do so in the future. The volatility and unpredictability of the price of virtual currency relative to fiat currency may result in significant loss over a short period of time. The nature of virtual currency may lead to an increased risk of fraud or cyber-attack. The nature of virtual currency means that any technological difficulties experienced by DMD may prevent the access or use of a customer’s virtual currency. Any bond or trust account maintained by Deangelo Michael Durden or any of its subsidiaries or affiliates for the benefit of its customers may not be sufficient to cover all losses incurred by customers.
TL;DR – The risk of loss in trading and holding cryptocurrencies can be substantial. You should carefully consider whether these practices make sense for you.
About Us

About[]()![InShot_20220420_162007165.jpg](https://user-images.githubusercontent.com/99563490/164339188-c82ab125-8629-480c-a16a-cee5f6973f14.jpg) Us
